packages/core/src/modules/action-menu/services/ActionMenuService.ts

```diff
@@ -346,6 +348,12 @@ export class ActionMenuService {
     })
   }
 
+  public async findByQuery(options: Query<ActionMenuRecord>) {
+    return await this.actionMenuRepository.findByQuery({
```

Comment:
Isn't possible to simply pass options as is, i.e. without the spread operator?

Reply:
It should, but I was thinking it maight be safer to pass in a spread object